### PR TITLE
fix typo around the project

### PR DESCRIPTION
**Pull request by cheyang**
Signed-off-by: cheyang <cheyang@163.com>

### CODE REVIEW

File: contributors/design-proposals/node/disk-accounting.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Each container and each image gets own block device. Since this driver works at
 
 Image layers and writable layers are stored under `/var/lib/docker/overlay/<id>`.
 
-Identical files are hardlinked between images.
+Identical files are hardline between images.
```
Comment (jbeda) [inline contributors/design-proposals/node/disk-accounting.md]: hardlinked is correct.

File: contributors/devel/component-config-conventions.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Configuration in general has poor qualities such as:
 
 1.  Configuration changes have the same forward/backward compatibility requirements as releases but rollout/rollback of configuration largely untested. Examples of configuration changes that might break a cluster: kubelet CNI plugin, etcd storage version.
 1.  Configuration options often exist only to test a specific feature where the default is reasonable for all real use cases. Examples: many sync periods.
-1.  Configuration options often exist to defer a "hard" design decision and to pay forward the "TODO(someonelse): think critically".
+1.  Configuration options often exist to defer a "hard" design decision and to pay forward the "TODO(someones): think critically".
```
Comment (jbeda) [inline contributors/devel/component-config-conventions.md]: This is most likely a person's github handle.

File: events/elections/2017/adnanabdulhussein_bio.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ the Kubernetes community. I think it's important that we uphold these values
 when defining and practicing a Code of Coduct and refining the contributor
 experience.
 
-Another area of the Steering Commitee I'm interested in is furthering the
+Another area of the Steering Committee I'm interested in is furthering the
```
Comment (jbeda) [inline events/elections/2017/adnanabdulhussein_bio.md]: This is a historical document. Please don't change this file.

**Comment by jbeda** [comment]
When submitting changes that are machine generated like this, please carefully review the changes.

**Comment by cblecker** [comment]
Please drop the change to `events/elections/2017/adnanabdulhussein_bio.md` as this is an historical file

File: events/elections/2017/adnanabdulhussein_bio.md (hunk 3 above)
Comment (cheyang) [reply events/elections/2017/adnanabdulhussein_bio.md]: I see. Thanks.

**Comment by cheyang** [comment]
@jbeda, Thanks for your suggestions.

**Pull request merged**